client/src/app/site/deployment-center/deployment-center-setup/step-configure/configure-vsts-build/windows-frameworks/windows-frameworks.component.ts

```diff
@@ -101,9 +103,27 @@ export class WindowsFramworksComponent implements OnInit, OnDestroy {
   }
   private setupValidators() {
     this.requiredValidator = new RequiredValidator(this._translateService, false);
+    const workingDirectoryValidator = RegexValidator.create(
+      new RegExp(/^(\.{2,}|~|())\\(.)*$|^(\.{2,}|~|())\/(.)*$|(.)+:(.)*/),
+      this._translateService.instant(PortalResources.validate_workingDirectory),
+      true
+    );
     this.wizard.buildSettings.get('applicationFramework').setValidators([this.requiredValidator.validate.bind(this.requiredValidator)]);
     this.wizard.buildSettings.get('applicationFramework').updateValueAndValidity();
 
+    this.wizard.buildSettings
+      .get('applicationFramework')
+      .valueChanges.takeUntil(this._ngUnsubscribe$)
+      .subscribe(stack => {
+        if (stack != WebAppFramework.AspNetCore && stack != WebAppFramework.AspNetWap) {
+          this.wizard.buildSettings.get('workingDirectory').setValidators([workingDirectoryValidator]);
+          this.wizard.buildSettings.get('workingDirectory').updateValueAndValidity();
+        } else {
+          this.wizard.buildSettings.get('workingDirectory').setValidators([]);
+          this.wizard.buildSettings.get('workingDirectory').updateValueAndValidity();
```

Comment:
you should also add this to remove validators function

Reply:
Done!